<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialogClass</class>
 <widget class="QDialog" name="AddItemDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddItemDialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>396</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <property name="verticalSpacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>选择要添加的媒体种类</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="mediaTypeBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>编号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="idLine"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>输入标题</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="titleLine"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>输入作者的名称</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="authorLine"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>选择评价分类等级</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="gradeBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>输入出版社</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="pressLine"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>输入ISBN</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="isbnLine"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>输入页数</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="pageLine"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>输入出品者名称</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="nameLine"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>出品年份</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="yearLine"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>视频时长</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="videoLengthLine"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>出品国籍</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="countryLine"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>图片长度</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lengthLine"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>图片宽度</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="widthLine"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="validText">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label</zorder>
   <zorder>mediaTypeBox</zorder>
   <zorder>idLine</zorder>
   <zorder>label_1</zorder>
   <zorder>label_2</zorder>
   <zorder>titleLine</zorder>
   <zorder>label_3</zorder>
   <zorder>authorLine</zorder>
   <zorder>label_4</zorder>
   <zorder>gradeBox</zorder>
   <zorder>label_5</zorder>
   <zorder>pressLine</zorder>
   <zorder>label_6</zorder>
   <zorder>isbnLine</zorder>
   <zorder>pageLine</zorder>
   <zorder>label_7</zorder>
   <zorder>nameLine</zorder>
   <zorder>label_8</zorder>
   <zorder>yearLine</zorder>
   <zorder>label_9</zorder>
   <zorder>videoLengthLine</zorder>
   <zorder>label_10</zorder>
   <zorder>countryLine</zorder>
   <zorder>label_11</zorder>
   <zorder>lengthLine</zorder>
   <zorder>label_12</zorder>
   <zorder>widthLine</zorder>
   <zorder>label_13</zorder>
   <zorder>validText</zorder>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
